index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Portfolio</title>
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="page-preview">
+      <nav>
+        <div class="left-logo">Gordon Otieno</div>
```

Comment:
- [ ] [REQUIRED] Please, make sure to put the logo in `<a>` element since this is more semantic. For most web users, they expect that the logo is always clickable.